assets/js/ModuleCategory.js

```diff
@@ -47,15 +52,13 @@ export default class ModuleCategory extends React.Component {
 
     var sortIcon = isOpen ? 'icon-sort-up-vl-gray' : 'icon-sort-down-vl-gray';
 
-    // ??? are these the correct icons per category ???
     const icons = {
       'Add data': 'database',
-      'Analyze': 'notepad',
+      'Analyse': 'notepad',         // Inconsistency with 'Analyze' elsewhere
       'Prepare':'wrangle',
       'Visualize': 'chart',
-      'Code': 'code',
+      'Code': 'code',               // not an active icon, needs replacement
```

Comment:
What do you mean "needs replacement"?

Reply:
Slight mixup there: I meant that there is no icon matching 'icon-code' in the CSS.

However, we do have the icon for 'code', but not for "Analyze/Analyse". (it looks like there are Modules in both categories)

Reply:
Have removed the deprecated module under the 'Analyse' category, all looks good now.